airbyte-workers/src/test-integration/java/io/airbyte/workers/process/AsyncOrchestratorPodProcessIntegrationTest.java

```diff
@@ -97,46 +98,51 @@ public static void init() throws Exception {
 
   @Test
   public void test() throws InterruptedException {
-    final var podName = "test-async-" + RandomStringUtils.randomAlphabetic(10).toLowerCase();
-
-    // make kubepodinfo
-    final var kubePodInfo = new KubePodInfo("default", podName);
-
-    // another activity issues the request to create the pod process -> here we'll just create it
-    final var asyncProcess = new AsyncOrchestratorPodProcess(
-        kubePodInfo,
-        documentStoreClient,
-        kubernetesClient,
-        null,
-        null,
-        "airbyte/container-orchestrator:dev",
-        null,
-        true);
-
-    final Map<Integer, Integer> portMap = Map.of(
-        WorkerApp.KUBE_HEARTBEAT_PORT, WorkerApp.KUBE_HEARTBEAT_PORT,
-        OrchestratorConstants.PORT1, OrchestratorConstants.PORT1,
-        OrchestratorConstants.PORT2, OrchestratorConstants.PORT2,
-        OrchestratorConstants.PORT3, OrchestratorConstants.PORT3,
-        OrchestratorConstants.PORT4, OrchestratorConstants.PORT4);
-
-    final Map<String, String> envMap = System.getenv().entrySet().stream()
-        .filter(entry -> OrchestratorConstants.ENV_VARS_TO_TRANSFER.contains(entry.getKey()))
-        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-
-    asyncProcess.create(Map.of(), new WorkerConfigs(new EnvConfigs()).getResourceRequirements(), Map.of(
-        OrchestratorConstants.INIT_FILE_APPLICATION, AsyncOrchestratorPodProcess.NO_OP,
-        OrchestratorConstants.INIT_FILE_ENV_MAP, Jsons.serialize(envMap)), portMap);
-
-    // a final activity waits until there is output from the kube pod process
-    asyncProcess.waitFor(10, TimeUnit.SECONDS);
-
-    final var exitValue = asyncProcess.exitValue();
-    final var output = asyncProcess.getOutput();
-
-    assertEquals(0, exitValue);
-    assertTrue(output.isPresent());
-    assertEquals("expected output", output.get());
+
+    final var pullPolicies = List.of("IfNotPresent", "Always");
```

Comment:
NIT: we should use 
```
@ParameterizedTest
@ValueSource(strings = {"IfNotPresent", " Always"})
```
instead of a loop.

Reply:
Agreed and made the modifications.